triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala

```diff
@@ -54,6 +56,11 @@ class Server(
 
   private var triggerLog: Map[UUID, Vector[(LocalDateTime, String)]] = Map.empty
 
+  private val authServiceClient = AuthServiceClient(InetAddress.getLoopbackAddress, Port(8089))(
```

Comment:
Host and port here need to be configurable and as mentioned above, I think a URI would be a better choice for an http endport.